src/main/java/fi/aalto/cs/apluscourses/presentation/module/ModuleListElementViewModel.java

```diff
@@ -87,13 +95,11 @@ public String getStatus() {
   }
 
   /**
-   * Returns a font weight in which the module is shown on a list.
-   * @return A {@link Float} that can be set to font weight.
+   * Indicates whether the module shown on a list should be displayed bold-faced.
    */
-  public float getFontWeight() {
+  @NotNull
+  public Boolean isBoldface() {
```

Comment:
a `Boolean` that is `@NotNull` makes me think there's no need for a 3-valued **boolean**, but why not the primitive then?